Configuration/ProcessModifiers/python/vertexInBlocks_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier enables, in the primary vertex producer,
+# - da clustering for PV in blocks
+vertexInBlocks =  cms.Modifier()
```

Comment:
I see a warning now about "no newline at the end file"
We ignore it?

Reply:
Restored.

Reply:
@mmusich this warning